src/patternfly/components/Card/card.scss

```diff
@@ -17,8 +17,17 @@
   --pf-c-card--m-compact--child--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-card--m-compact--c-divider--child--PaddingTop: var(--pf-global--spacer--md);
   --pf-c-card--m-compact__title--not--last-child--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-card--m-display-lg__body--FontSize: var(--pf-global--FontSize--lg);
+  --pf-c-card--m-display-lg__footer--FontSize: var(--pf-global--FontSize--lg);
+  --pf-c-card--m-display-lg--first-child--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-card--m-display-lg--child--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-card--m-display-lg--child--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-card--m-display-lg--child--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-card--m-display-lg--c-divider--child--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-card--m-display-lg__title--not--last-child--PaddingBottom: var(--pf-global--spacer--xl);
```

Comment:
```suggestion
  --pf-c-card--m-display-lg__title--not--last-child--PaddingBottom: var(--pf-global--spacer--lg);
```

Reply:
@DaoDaoNoCode confirmed with @mceledonia that everything should have an `xl` spacer except the space below the title, that should be `lg`